backend/cabin-sorting/hackerSortingAlgo.ts

```diff
@@ -30,6 +31,13 @@ function loadCSV (filepath: string, headers: boolean): any[] {
   return parse(fileContent, options)
 }
 
+async function loadFromDatabase (): Promise<any[]> {
+    const response = await axios.get("http://localhost:4000/sortedHackers")
+    const data = response.data;
+    console.log(data);
```

Comment:
is this console log for debugging or does it output useful information for the user/should this be parsed to be more user friendly